data/asia.yaml

```diff
@@ -124,3 +124,9 @@
    long: 126.8917324
    lat: 37.4769042
    link: http://www.robotis.com/
+ - name: Tectus Dreamlab Pte Ltd
+   type: company
+   address: 1 Fusionopolis Way, Connexis South Tower #20-03, Singapore 138632
```

Comment:
Just to be on the safe side, can you quote this string? Otherwise the last bit may get left off

Reply:
Hi. I just made the changes.

It seems like no one else is quoting the string. So I thought it's unnecessary 😄